bricks/vs/src/code-editor/index.tsx

```diff
@@ -638,18 +682,67 @@ export function CodeEditorComponent({
     };
   }, []);
 
+  const handleCopyIconClick = useCallback(() => {
+    if (editorRef.current) {
+      const currentModel = editorRef.current.getModel()!;
+      copyToClipboard(currentModel.getValue())
+        .then(() =>
+          showNotification({ type: "success", message: t(K.COPY_SUCCESS) })
+        )
+        .catch(() =>
+          showNotification({ type: "error", message: t(K.COPY_FAILED) })
+        );
+    }
+  }, [t]);
+
+  const handleExpandedClick = useCallback(() => {
+    setExpanded(!expanded);
+  }, [expanded]);
+
   return (
     <div
-      ref={containerRef}
-      style={{
-        height: actualHeight,
-        ...(validateState === "error"
-          ? {
-              outline: "1px solid var(--antd-error-color)",
-            }
-          : {}),
-      }}
-    />
+      className={classNames("code-editor-wrapper", {
+        expanded,
+      })}
+    >
+      <div
+        ref={containerRef}
+        style={{
+          height: expanded ? "100%" : actualHeight,
+          ...(validateState === "error"
+            ? {
+                outline: "1px solid var(--antd-error-color)",
+              }
+            : {}),
+        }}
+      />
+      <div className="toolbar">
+        {showCopyButton && (
+          <WrappedTooltip content={t(K.COPY) as string}>
+            <WrappedIcon
+              className="copy-icon"
+              icon="copy"
+              lib="antd"
+              theme="outlined"
+              onClick={handleCopyIconClick}
+            />
+          </WrappedTooltip>
+        )}
+        {showExpandButton && (
+          <WrappedTooltip
+            content={(expanded ? t(K.SHRINK) : t(K.EXPAND)) as string}
+          >
+            <WrappedIcon
+              className="expand-icon"
+              icon={expanded ? "shrink" : "expand"}
```

Comment:
Antd 的 expand 那个图标和 compress 对应

```suggestion
              icon={expanded ? "compress" : "expand"}
```